pkg/domain/attributor.go

```diff
@@ -99,46 +98,50 @@ func (n *CloudAssetDetails) UnmarshalJSON(data []byte) error {
 // returns either a 404 Not Found response, or a 200 OK response with no results
 type AssetNotFoundError struct {
 	Inner          error
-	AssetID        string
-	ScanTimestamp  string
 	AssetInventory string
+	Code           int
+	ScanTimestamp  string
 }
 
 func (err AssetNotFoundError) Error() string {
-	return fmt.Sprintf(
-		"Result not found for asset with ID %s as of scan time %s in asset inventory %s: %v",
-		err.AssetID, err.ScanTimestamp, err.AssetInventory, err.Inner)
+	return "Result not found for asset using asset inventory"
 }
 
 // AssetInventoryRequestError occurs when a request to an asset inventory system
 // returns a failure response
 type AssetInventoryRequestError struct {
 	Inner          error
-	AssetID        string
-	ScanTimestamp  string
 	AssetInventory string
 	Code           int
+	ScanTimestamp  string
 }
 
 func (err AssetInventoryRequestError) Error() string {
-	return fmt.Sprintf(
-		"Request to asset inventory %s failed with code %d for asset with ID %s as of scan time %s: %v",
-		err.AssetInventory, err.Code, err.AssetID, err.ScanTimestamp, err.Inner)
+	return "Request to asset inventory for asset"
 }
 
 // AssetInventoryMultipleAssetsFoundError occurs when a request to an asset inventory system
 // returns a successful response with multiple assets
 type AssetInventoryMultipleAssetsFoundError struct {
 	Inner          error
-	AssetID        string
-	ScanTimestamp  string
 	AssetInventory string
+	ScanTimestamp  string
 }
 
 func (err AssetInventoryMultipleAssetsFoundError) Error() string {
-	return fmt.Sprintf(
-		"Request to asset inventory %s returned multiple values for asset with ID %s as of scan time %s: %v",
-		err.AssetInventory, err.AssetID, err.ScanTimestamp, err.Inner)
+	return "Request to asset inventory %s returned multiple values for asset"
```

Comment:
Task: remove the unused formatting directive here (`%s`).
Suggestion: make "Request" lowercase; otherwise, I like the text here.